### DISCUSSION TITLE
Criteria for donating a project to the GSF

### DISCUSSION THREAD

**Discussion by jawache** [Green-Software-Foundation/oc#22] [General]
Hia, I can see conversations around this topic in other threads so I think let's deal with it directly.

Our goal is to build a trusted ecosystem, there is lots out there in the world, if the GSF says something is good, you can trust it's good.

When we have a **graduated** project in our repo we are giving the world confidence that they can trust that project, it has a community around it, it will be supported, bugs will be resolved. They can use it in production. 

The bar for a project to be **donated** to the Foundation should be very high, I propose they have to meet the **graduated** status to be a GSF project. 

In addition to meeting the requirements of a graduated project, to be accepted into the GSF a project should have:

- A **strong** commitment from member orgs to commit resources to work on that project. (I think this should have to go beyond casual volunteering, there needs to be resources and hours committed to a project by orgs for us to be able to support it properly)
- A **strong** commitment from at least one individual to lead the project. 
- A willingness on the part of the donor to give up maintenance rights on the project if they are not also a GSF member, only members of the GSF can lead projects or have any form of meaningful permissions on GitHub.

Without at least the first two above there really isn't much reason to accept a project into the Foundation. 

From my experience of other Foundations what I've seen happen is a member org has donated a project they have worked on to the Foundation. The member org continues to work on the project and continues to commit resources to it, however the governance moved to a collective (GSF) rather than one company which can help an open source project evolve and give people more confidence in it. 

We should explore more lighter weight mechanisms of supporting external projects, endorsing them, co-promoting them, collaborations things like that. But bringing it into the Foundation should be very special, meaningful and rare IMO.

**Comment by Sealjay** [comment]
https://github.com/Green-Software-Foundation/oc/issues/25
@jawache 
We agreed with your proposal.

**Comment by Sophietn** [comment]
I have factored in this comment into my proposal for new process and new criteria. 
Captured in my comment on [this issue](https://github.com/Green-Software-Foundation/oc/issues/25) and PRs for review. 
I'll close this discussion.
